Various nits and wordsmithing (#654)
ietf-tapswg/api-drafts: draft-ietf-taps-interface.md

Boundaries, but there is a transport protocol that provides a reliable ordered
byte stream, an application could receive this byte stream as partial
Messages and transform it into application-layer Messages.  Alternatively,

> an application might provide a Message Framer, which transforms a

Reviewer: ```suggestion
an application might provide a Message Framer, which can transform a
```
I suggest to keep the "can transform" because I don't think Message Framers *always* do this transformation.

> byte stream to or from a sequence of Messages ({{framing}}).

Reviewer: ```suggestion
sequence of Messages into a byte stream and vice versa ({{framing}}).
```

I like your suggestion better than what we have (the original just maps better to why this is really necessary in our own minds IMO), but now it seems awkward to begin with the byte stream... I just think that it reads a bit nicer with the change above.

~~~

- Session cache management: Used to tune cache capacity, lifetime, re-use,

> and eviction policies, e.g., LRU or FIFO.may also be changed, but are implementation-specific.

Reviewer: ```suggestion
and eviction policies, e.g., LRU or FIFO.
```

I agree with your proposed change, but this missed something more - the sentence was broken ("FIFO.may ..."). ISTM that this extra sentence is not needed here anyway.
